src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      data-qa="big" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get"
+    >
+      <label>
+        <img 
+          class="iconL" 
+          src="images/Search.svg" 
+          alt="search icon"
+        >
+        <input 
+          data-qa="keypress" 
+          class="input big" 
```

Comment:
please follow approach from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p2 -  combine properties together and then override necessary properties for small input 


```suggestion
          class="input" 
```